docs/best_practices/tool_xml.rst

```diff
@@ -88,6 +88,14 @@ would have a hard time finding it and gathering context about its functionality.
 With the VSearch prefix, once a user learns what one VSearch tool does, they can
 quickly apply that to the other available VSearch tools.
 
+Tool profile
+------------
+
+Tools should define a recent profile:
+
+- for new tools IUC requires the most recent Galaxy release
+- tool updates should try to use a profile not older than 2 years. 
```

Comment:
What about requiring a supported Galaxy release (i.e. not older than a year) for both?

Reply:
While well intended, a very strict policy here will hamper tool authors with an older local Galaxy server. I would therefore keep this as "should" not "must".

Reply:
> While well intended, a very strict policy here will hamper tool authors with an older local Galaxy server. I would therefore keep this as "should" not "must".

I agree. So "should" + 1 year in both cases?